feat: generate tests with env files (#2230)
kubeshop/testkube: docs/generate-test-crds.md

> # File naming convention for test CRD generation

Reviewer: ```suggestion
# File Naming Convention for Test CRD Generation
```

> We support following file naming convention for Postman files to support multple environment files:

Reviewer: ```suggestion
We support the following file naming convention for Postman files to support multiple environment files:
```

> ## Test filename convention

Reviewer: ```suggestion
## Test Filename Convention
```

> <Test name>.postman_collection.json - where `Test name` should be reused for environments files.

Reviewer: ```suggestion
<Test name>.postman_collection.json - where `Test name` should be reused for the environment files.
```

> For example, mytest.postman_collection.json

Reviewer: ```suggestion
For example, mytest.postman_collection.json.
```

> ## Test environment filename convention

Reviewer: ```suggestion
## Test Environment Filename Convention
```

> ## Test environment filename convention

Reviewer: ```suggestion
## Test Environment File Naming Convention
```

> <Test name>.<Test env>.postman_environment.json - where `Test name` is reused from test files and 

Reviewer: ```suggestion
<Test name>.<Test env>.postman_environment.json - where `Test name` is reused from the test files and 
```

> `Test env` is pointing to particular testing environment.

Reviewer: ```suggestion
`Test env` is pointing to a particular testing environment.
```

> For example, mytest.prod.postman_collection.json

Reviewer: ```suggestion
For example, mytest.prod.postman_collection.json.
```

> ## Test secret environment filename convention

Reviewer: ```suggestion
## Test Secret Environment File Naming Convention
```

<Test name>.<Test env>.postman_secret_environment.json - where `Test name` is reused from test files and 
`Test env` is pointing to particular testing environment.

> For example, mytest.prod.postman_secret_environment.json

Reviewer: ```suggestion
For example, mytest.prod.postman_secret_environment.json.
```
